From 7284fd1a0c25d4038f95cdfceee93398f324b7ae Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Wed, 21 Nov 2012 13:56:49 +0530
Subject: Enable WMA9 audio Decoding support

BZ: 50306

This patch contains changes in the libstagefright
to enable audio decoding capabilities

Category:device enablement
Domain: AUDIO_MEDIA-Audio Codecs
Origin: internal
Upstream-Candidate: no

Change-Id: I18137307c996a0e64f56a03905f4cd932e625b44
Orig-Change-Id: I3861958c7142795dc68829117a1751a7fad113d3
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
Signed-off-by: Dan Liang <dan.liang@intel.com>
---
 include/media/stagefright/MetaData.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/include/media/stagefright/MetaData.h b/include/media/stagefright/MetaData.h
index de3fc36..92f2370 100644
--- a/include/media/stagefright/MetaData.h
+++ b/include/media/stagefright/MetaData.h
@@ -161,6 +161,12 @@ enum {
     kKeyCryptoDefaultIVSize = 'cryS',  // int32_t
 
     kKeyPssh              = 'pssh',  // raw data
+#ifdef USE_INTEL_ASF_EXTRACTOR
+    kKeyWmaFormatTag      = 'frmt',
+    kKeyBitPerSample      = 'btps',
+    kKeyWmaBlockAlign     = 'blka',
+#endif //ifdef USE_INTEL_ASF_EXTRACTOR
+
 #ifdef QCOM_HARDWARE
     kKeyTunnelException   = 'Ntnl', // not tunnel
 #endif
-- 
1.7.4.1

